### DISCUSSION TITLE
Burning the eFuses

### DISCUSSION THREAD

**Discussion by vaderag** [darthcloud/BlueRetro#44] [Q&A] [answered]
So, I've got the following moduld so believe i need to "Burn the fuses"
![image](https://user-images.githubusercontent.com/17175988/123127045-d2d3f100-d441-11eb-89a8-3f70e9975b5c.png)

I'm looking at the instructions here but I'm not quite sure where I'm supposed to enter the commands - can someone advise?
https://docs.espressif.com/projects/esp-idf/en/latest/esp32/api-reference/peripherals/sd_pullup_requirements.html#strapping-conflicts-dat2

Also, I want to make sure this is necessary - seems this _may_ be related to SD card (if I'm reading correctly) and I'm not using an SD...

Thanks!

**Comment by darthcloud** [comment, marked as answer]
You dont need, its SD only
